.github/workflows/build.yml

```diff
@@ -1,14 +1,21 @@
 name: Build Workflow
 
 on:
-  pull_request:
   push:
-
+   branches:
+     - 'master'
+     - 'develop'
+     - 'release/*'
+  pull_request:
+      types: [opened, synchronize, reopened]
 jobs:
   build-package:
     strategy:
       matrix:
         include:
+          - architecture: x64
```

Comment:
this is not referring to the package we build right? which may have carma-system-4.5.0 prefix now?